.github/workflows/build.yml

```
@@ -13,35 +13,76 @@ jobs:
+        uses: actions/setup-node@v4
+        with:
+          node-version: ${{ env.NODE_VERSION }}
+          cache: 'npm'
+      - run: npm ci
+      - run: npm run build
+      - name: Checkout JAM test vectors
+        uses: actions/checkout@v4
+        with:
+          repository: FluffyLabs/jamtestvectors
+          path: "./jamtestvectors"
+          ref: ba76542dbf7a0c72d414a87ad2e30ce4da380448 # New test vectors.
```

YAMLlint (1.38.0)

[warning] too few spaces before comment: expected 2

(comments)